### DISCUSSION TITLE
To add records count to filters

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16735] [Feature improvement]
Adding records count (the same way as related lists) to filters will improve control over number of filtered records. For instance Helpdesk. System converting emails to tickets, reassigning them automatically and or setting relevant statuses. 
Having clear overview on number of records without clicking each filter will improve accessibility performance.

Best

**Comment by rskrzypczak** [comment]
There is already something like this, but only for filters in the menu.
![image](https://user-images.githubusercontent.com/10221999/207524670-fe6c53b2-e8d1-4d47-ac83-5d0ff2ce0e7e.png)
![image](https://user-images.githubusercontent.com/10221999/207524814-6d2cd3ad-72fc-4c39-9184-b8b36da638b2.png)
